tests/python_package_test/test_basic.py

```diff
@@ -217,6 +217,43 @@ def test_sequence_get_data(num_seq):
     np.testing.assert_array_equal(subset_data.get_data(), X[sorted(used_indices)])
 
 
+def test_retrain_list_of_sequence():
+    X, y = load_breast_cancer(return_X_y=True)
+    seqs = _create_sequence_from_ndarray(X, 2, 100)
+
+    seq_ds = lgb.Dataset(seqs, label=y, free_raw_data=False)
```

Comment:
Why was `free_raw_data=False` necessary here? If it wasn't, please remove it.

Reply:
If free_raw_data=True, model2 cannot get the data, would raise Exception I remember .

Reply:
Ok, I will push testing changes for you.